.zenodo.json

```diff
@@ -3,28 +3,99 @@
   "description": "The alpaka library is a header-only C++14 abstraction library for accelerator development. Its aim is to provide performance portability across accelerators through the abstraction (not hiding!) of the underlying levels of parallelism.",
   "creators": [
     {
-      "affiliation": "LogMeIn, Inc.",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden, LogMeIn Inc.",
       "name": "Worpitz, Benjamin"
     },
     {
-      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden",
-      "name": "Matthes, Alexander",
-      "orcid": "0000-0002-6702-2015"
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+      "name": "Widera, René",
+      "orcid": "0000-0003-1642-0459"
     },
     {
-      "affiliation": "LogMeIn, Inc.",
-      "name": "Zenker, Erik",
-      "orcid": "0000-0001-9417-8712"
+      "name": "Bastrakov, Sergei",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+      "orcid": "0000-0003-3396-6154"
+    },
+    {
+      "name": "Colgrove, Mat",
+      "affiliation": "NVIDIA"
+    },
+    {
+      "name": "Ehrig, Simeon",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+      "orcid": "0000-0002-8218-3116"
+    },
+    {
+      "name": "Gruber, Bernhard Manfred",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, CASUS, CERN",
+      "orcid": "0000-0001-7848-1690"
+    },
+    {
+      "name": "Kelling, Jeffrey",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf",
+      "orcid": "0000-0003-1761-2591"
+    },
+    {
+      "name": "Krude, Jakob",
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf"
+    },
+    {
+      "affiliation": "Helmholtz-Zentrum Dresden-Rossendorf, CASUS",
```

Comment:
Do we need a separate CASUS affiliation? We are a subdivision of HZDR.

Reply:
Idk, maybe ask Michael which way he likes?

Reply:
I think keeping CASUS separate should be OK, because it is only managed by HZDR, I will ping Michael.